Controllers/OpinionManagement.php

```diff
@@ -3,16 +3,21 @@
 namespace OpinionManagement\Controllers;
 
 use Doctrine\Common\Collections\Criteria;
+use EvaluationMethodTechnical\Plugin as EvaluationTechnicalPlugin;
 use MapasCulturais\Controller,
     MapasCulturais\App;
+use MapasCulturais\Entities\Opportunity;
+use MapasCulturais\Entities\EvaluationMethodConfigurationMeta;
 use MapasCulturais\Entities\Notification;
+use MapasCulturais\Entities\Registration;
+use MapasCulturais\Entities\RegistrationEvaluation;
 use OpinionManagement\Helpers\EvaluationList;
 
 class OpinionManagement extends Controller
 {
     public function GET_index(): void
     {
-        $app = App::i();
+        $app = (new App)->i();
```

Comment:
Não seria bom evitar essa forma de instanciar? Por que dessa forma, toda vez que o método index for chamado terá uma nova instancia, a forma anterior só instancia um única fez mantendo o padrão criacional do Mapa.